src/reducer.js

```diff
@@ -0,0 +1,34 @@
+const initialState = {
+  regions: [],
+  categories: [],
+};
+
+const reducers = {
+  setRegions(state, action) {
+    const { regions } = action.payload;
```

Comment:
```suggestion
  setRegions(state, { payload: { regions } }) {
```
좀 더 심플하게 쓸 수도 있을 것 같아요~